src/utils/Git.ts

```diff
@@ -127,16 +126,22 @@ export async function branches({
         ['"symbolic-ref"', '"--short"', '"-q"', '"HEAD"'],
         directory)
         .then(branchName => branchName.trim());
+    const branchFilter = [
+        "s/^refs\\/heads\\///g",
+        remotes ? "s/^refs\\/remotes\\/[^\\?%\\*\\:\\|\"\\<\\>\\\\]*\\///g" : "",
+        tags ? "s/^refs\\/tags\\///g" : ""].join(";");
```

Comment:
Thanks good tip! I've made some code updates but since the repo rename to "upterm" and some other commits branch auto-complete suggestions for "git checkout" don't seem to show for me now. I'll have another look in the next few days.